<launch>
    <arg name="camera_name" default="sony_cam1"/>
    <arg name="camera_id" default="0"/>
    <arg name="device_ip" default="192.168.1.201"/>
    <arg name="camera_fps" default="60"/>
    <arg name="camera_calib" default="file://$(find sony_cam)/config/$(arg camera_name)_info.yaml"/>
    <arg name="image_width" default="640"/>
    <arg name="image_height" default="480"/>

    <node name="$(arg camera_name)"
        pkg="sony_cam"
        type="sony_cam"
        output="screen">
    <param name="frame_rate" value="$(arg camera_fps)"/>
    <param name="camera_name" value="$(arg camera_name)"/>
    <param name="camera_frame_id" value="$(arg camera_name)"/>
    <param name="device_id" value="/dev/video$(arg camera_id)"/>
    <param name="image_width" value="$(arg image_width)"/>
    <param name="image_height" value="$(arg image_height)"/>
    <param name="camera_info_url" value="$(arg camera_calib)"/>
    </node>
</launch>